presto-native-execution/presto_cpp/main/PrestoServer.h

```diff
@@ -124,6 +124,8 @@ class PrestoServer {
   /// In the implementation any extra config properties can be registered.
   virtual void registerExtraConfigProperties() {}
 
+  virtual std::string getLocalIp() const;
```

Comment:
It would be nice to document this method and explain the reason for exposing it as a virtual method.